site/docs/composition.md

```diff
@@ -5,4 +5,42 @@ title: Composing Layered & Multi-view Plots
 permalink: /docs/composition.html
 ---
 
-TODO Intro to composition and link to compositional operators
+With Vega-Lite, you cannot only create visualizations that consist of a single view. With Vega-Lite's *view composition algebra*, you can combine views into larger multiview displays. You can combine views through [faceting](facet.html), [layering](layer.html), [concatenation](concat.html), and [repeating](repeat.html).
```

Comment:
"you cannot only" -> "you can not only". It's also missing a "but also" clause. Perhaps the following:

> With Vega-Lite, you can not only create single view visualizations, but also [facet], [layer], [concatenate], and [repeat] these views into larger, more complex multiview displays.

(I think we should consider dropping "algebra" language from the documentation. That seems fairly complex for docs).